### ISSUE TITLE
It's not possible to close pop-up windows using the ESC key

### ISSUE THREAD

**Issue by annirytkonen** [apluslms/a-plus#1013]
It should be possible to close the pop-up windows using the ESC key, like on most other websites.

**Comment by Mankro** [comment]
For students, this is very prominent on the chapter pages and the submission feedback modal dialogues. Currently, it is not possible to close the modal with the ESC key.

We already had the idea that the whole submission modal dialog should be removed from chapter pages. This requires a revamp of the chapter/exercise user interface.
* https://github.com/apluslms/a-plus/issues/589#issuecomment-649481522
* https://github.com/apluslms/a-plus/issues/492
* https://github.com/apluslms/a-plus/issues/494

If modal dialogs stay, related issues: #589, https://apluslms.github.io/accessibility-audit/patterns/fixable-a/modals/
* hard to move the focus to the modal content with the TAB key
* can not exit the modal with the ESC key

**Comment by Mankro** [comment]
There is "interesting" behavior when you open two modals at the same time. In the "Your points" page (and also in the chapter pages), you can open submission feedback in a modal dialog. In the modal, you can also open another modal for the submitted file (it shows the file contents, the submitted source code in programming assignments). Now with two modals open, if you press the ESC key, it actually closes the feedback modal, but leaves the submitted file modal open. In addition, scrolling the page with the mouse wheel moves the actual page at this point instead of the modal contents.

A student has also reported that when you have those two modals open, if you click with the mouse in the side of the page outside the modal, it closes the feedback modal but leaves the submitted file modal open. However, I am not able to reproduce this myself on Firefox. For me, clicking in the side closes the submitted file modal first and another click closes the feedback modal. The mouse wheel scrolling issue starts after closing one modal, that is, you can't scroll the modal contents anymore.
